### PR TITLE
Yuning hosting events

### PR DESCRIPTION
**Pull request by ynzhang09**
For this PR, the feature of officers adding events to the event page was added. 

User Story: https://mchso-infoorganizer.atlassian.net/browse/MCHSO-17

Acceptance Criteria:
- An officer can add events from the event page
- The event requires a name, description, and number of points.

Definition of Done:
- Revied by the product owner
- 100% code coverage
- Satisfies acceptance criteria
- Deployed on website

### CODE REVIEW

**Comment by jswood23** [comment]
Are we gonna show upcoming events in a table? It seems like it would be better to list them in individual sections or cards.
Also do we want to show the points for each event on the external events page?

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,15 @@
 Rails.application.routes.draw do
+  # get 'events/index'
+  # get 'events/show'
+  # get 'events/new'
+  # get 'events/edit'
+  # get 'events/delete'
```
Comment (jswood23) [inline config/routes.rb]: Let's clean this up so the routing for the events pages is uniform with the rest of the file.

File: app/views/events/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @events.each do |event| %>
+      <tr>
+        <td><%= event.event_name %></td>
+        <td><%= event.description %></td>
+        <td><%= event.date %></td>
+        <td><%= event.points %></td>
```
Comment (jswood23) [inline app/views/events/index.html.erb]: Should we be showing points here?

File: spec/helpers/events_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+#
+# describe EventsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe EventsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (jswood23) [inline spec/helpers/events_helper_spec.rb]: This needs to be either commented out or replaced with an integration test.

File: spec/views/events/delete.html.erb_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "events/delete.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (jswood23) [inline spec/views/events/delete.html.erb_spec.rb]: Same here

File: spec/views/events/edit.html.erb_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "events/edit.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (jswood23) [inline spec/views/events/edit.html.erb_spec.rb]: And here

File: spec/views/events/new.html.erb_spec.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "events/new.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (jswood23) [inline spec/views/events/new.html.erb_spec.rb]: And here

File: spec/views/events/show.html.erb_spec.rb [hunk 7]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "events/show.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (jswood23) [inline spec/views/events/show.html.erb_spec.rb]: And here

**Review by jswood23 — CHANGES_REQUESTED** [review]

File: spec/views/events/new.html.erb_spec.rb (hunk 6 above)
Comment (ynzhang09) [reply spec/views/events/new.html.erb_spec.rb]: Added comment

File: spec/views/events/show.html.erb_spec.rb (hunk 7 above)
Comment (ynzhang09) [reply spec/views/events/show.html.erb_spec.rb]: Added comment

File: spec/views/events/edit.html.erb_spec.rb (hunk 5 above)
Comment (ynzhang09) [reply spec/views/events/edit.html.erb_spec.rb]: Added comment

File: app/views/events/index.html.erb (hunk 2 above)
Comment (MengtingCao) [reply app/views/events/index.html.erb]: We can show the members how many points an event is worth.

**Review by MengtingCao — CHANGES_REQUESTED** [review]
Should we close this PR?

**Pull request closed without merging**